htdocs/core/ajax/ajaxfield.php

```diff
@@ -0,0 +1,150 @@
+<?php
```

Comment:
Is it a file for the webportal ? If yes, we should find it into public/webportal/... otherwise, when webportal will be behind its own virtual host name, the file will not be reachable as on file into subdir public/webportal are files the browser can connect to.

Also, as it is a page, we shoul dhave a protection to disable it when module webportal is off.

The include should be webportal.main.inc.php because this main may use a different policy for cookie and security http headers. The main.inc.php is reserved for the security inside the backoffice that is a different environment than the one of the webportal.